examples/browser/camera/IE-ES5-RTSP-simple-stream/rtsp-player.js

```diff
@@ -0,0 +1,83 @@
+(function () {
```

Comment:
This file fails to pass the unit tests (prettier formatting). Please run `yarn prettier:fix` to properly format all files so that the PR passes the tests.

Reply:
Thanks! As soon as I find some extra time I will make the changes. 
Cheers.